changelog.d/20240520_150131_danyal.faheem_upgrade_mysql_utf8mb4.md

```diff
@@ -0,0 +1,2 @@
+- [Feature] Upgrade default charset and collation of mysql to utf8mb4 and utf8mb4_unicode_ci respectively (by @Danyal-Faheem)
```

Comment:
nit: this is a breaking change in a capacity, lets add 💥 in changelog to ensure that.

Reply:
Done!